@here/harp-mapview/lib/DecodedTileHelpers.ts

```diff
@@ -457,26 +458,49 @@ export function applyTechniqueToMaterial(
     level?: number,
     skipExtraProps?: string[]
 ) {
-    Object.getOwnPropertyNames(technique).forEach(propertyName => {
+    const commonProps = Object.getOwnPropertyNames(technique).filter(propertyName => {
         if (
             propertyName.startsWith("_") ||
```

Comment:
`transparentColor` (if exists) and `opacity` and ``transparent` should be skipped here IMO

Reply:
This can be done, but with some changes to the logic, imagine that technique has transparent color attribute (in descriptor), this color is defined, but does not contain the transparency octet (alpha value), then with current implementation material.opacity & material.transparent will never be defined (transferred from technique to material).
Of course we may do this, with really small overhead: 2 number multiplications regardless of the color defined.

Reply:
Ok, never mind, I have changed logic/flow a lot, so transparent/opacity material properties are now removed from the first stage and applied with _main/base_ color if any, otherwise nothing is change and all props are applied as usually.